wasm/src/main/java/com/dylibso/chicory/wasm/Parser.java

```diff
@@ -308,13 +309,39 @@ private static ImportSection parseImportSection(
             var descType = ImportDescType.byId(readVarUInt32(buffer));
             switch (descType) {
                 case FuncIdx:
-                    var funcDesc = new ImportDesc(descType, (int) readVarUInt32(buffer));
-                    imports[i] = new Import(moduleName, fieldName, funcDesc);
-                    break;
+                    {
+                        var funcDesc = new ImportDesc(descType, (int) readVarUInt32(buffer));
+                        imports[i] = new Import(moduleName, fieldName, funcDesc);
+                        break;
+                    }
                 case TableIdx:
-                    throw new ChicoryException("Don't support table type globals yet");
+                    {
+                        var min = (int) readVarUInt32(buffer);
+                        var next = (int) readVarUInt32(buffer);
+                        var max = 0;
+                        if (buffer.hasRemaining()) {
```

Comment:
I'm not 100% this is the right way to handle the "optional" max value, I welcome any idea.

Reply:
found the right encoding